test/unit_tests/conversation/ConversationRepositorySpec.coffee

```diff
@@ -288,6 +288,91 @@ describe 'z.conversation.ConversationRepository', ->
       result = conversation_repository.get_groups_by_name 'Removed'
       expect(result.length).toBe 0
 
+  describe 'delete_message_everyone', ->
+
+    conversation_et = null
+
+    beforeEach ->
+      conversation_et = _generate_conversation z.conversation.ConversationType.REGULAR
+
+      spyOn(conversation_repository, '_send_encrypted_value').and.returnValue Promise.resolve()
+
+    it 'does not delete other users messages', (done) ->
+      user_et = new z.entity.User()
+      user_et.is_me = false
+      message_to_delete_et = new z.entity.Message()
+      message_to_delete_et.id = z.util.create_random_uuid()
+      message_to_delete_et.user user_et
+      conversation_et.add_message message_to_delete_et
+
+      conversation_repository.delete_message_everyone conversation_et, message_to_delete_et
+      .then done.fail
+      .catch done
+
+    it 'sends delete and deletes message for own messages', (done) ->
+      user_et = new z.entity.User()
+      user_et.is_me = true
+      message_to_delete_et = new z.entity.Message()
+      message_to_delete_et.id = z.util.create_random_uuid()
+      message_to_delete_et.user user_et
+      conversation_et.add_message message_to_delete_et
+
+      expect(conversation_et.get_message_by_id(message_to_delete_et.id)).toBeDefined()
```

Comment:
Parenthesis not needed here or in many of the following similar lines: `expect(conversation_et.get_message_by_id message_to_delete_et.id).toBeDefined()`